Data-Structures/Linked-List/SinglyLinkedList.js

```diff
@@ -274,6 +274,20 @@ class LinkedList {
   log () {
     console.log(JSON.stringify(this.headNode, null, 2))
   }
+
+  // Method to reverse the LinkedList where each node contains a single integer value
```

Comment:
It doesn't matter what the nodes contain

Reply:
Corrected.